jsonnet/telemeter/rules.libsonnet

```diff
@@ -196,23 +196,30 @@
                 (sum by (_id, tenant_id) (cluster:capacity_cpu_cores:sum{label_node_openshift_io_os_id="rhcos",label_node_role_kubernetes_io="master",label_kubernetes_io_arch!="amd64"}) * on(_id, tenant_id) group by(_id, tenant_id) (cluster_master_schedulable == 1) or cluster:cpu_capacity_cores:_id)
               |||,
             },
+            {
+              // Aggregate managed nodes under the cluster so swatch can associate the capacity with the cluster's billing account information
+              record: 'acm_capacity_effective_cpu_cores:sum',
+              expr: |||
+                sum by (_id) (acm_capacity_effective_cpu_cores)
+              |||,
+            },
```

Comment:
it needs to come after the definition of `acm_capacity_effective_cpu_cores`.

Reply:
moved, thanks!